packages/zoltan2/example/geometric/rcb_C.cpp

```diff
@@ -186,14 +185,14 @@ int main(int argc, char *argv[])
 
   // Create a Zoltan2 input adapter that includes weights.
 
-  vector<const scalar_t *>coordVec(2);
-  vector<int> coordStrides(2);
+  std::vector<const scalar_t *>coordVec(2);
```

Comment:
It's OK to have `using namespace std;` at function scope, just not at global scope.  Also, you still have `using std::vector` above (which is fine, it's just that you didn't need to make this change in that case).